<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurePlayer</class>
 <widget class="QWidget" name="ConfigurePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	background-color:qlineargradient( x1:0.3, y1:0.3, x2:0.1, y2:0.99, 
									stop: 0.1 rgb(100, 199, 154),
									stop:0.4 rgb(90, 195, 75),
									stop:1 rgb(207, 255, 174));
	border:1px solid rgb(143, 172, 175);
	border-radius:0;
}

#resetButton{
	background-color:qlineargradient( x1:0.1, y1:0.4, x2:0.3, y2:0.9, 
									stop: 0.1 rgb(199, 99, 109),
									stop:0.4 rgb(209, 21, 99),
									stop:1 rgb(214, 115, 154));
}

QPushButton:hover,
#resetButton:hover
{
	background-color:qlineargradient( x1:0.1, y1:0.5, x2:1, y2:0.9, 
									stop:0 rgb(242, 199, 161), 
									stop:0.5 rgb(209, 115, 55), 
									stop:1 rgb(234, 255, 255));
}


QPushButton:pressed,
#resetButton:pressed
{
	background-color:qlineargradient( x1:0.1, y1:0.4, x2:0.3, y2:0.9, 
									stop: 0.1 rgb(219, 126, 109),
									stop:0.4 rgb(202, 115, 54),
									stop:1 rgb(182, 115, 194));
	border:1px solid rgb(143, 172, 175);
}

</string>
  </property>
  <widget class="QCommandLinkButton" name="selectPictBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton{
	font-size:11px;
	color: rgb(0, 27, 40);
}</string>
   </property>
   <property name="text">
    <string>Change picture</string>
   </property>
   <property name="description">
    <string>Select a picture in your Pc</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>21</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="title">
    <string>Edit group and Players name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="groupNameEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Name of the group</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="player1_Name">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Name for player 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="player2_Name">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Name for player 2</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Save change</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="discardBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Discard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCommandLinkButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton{
	font-size:11px;
	color: rgb(0, 27, 40);
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="description">
    <string>&amp;Reset values to default</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
